patches/minecraft/net/minecraft/tileentity/TileEntityFurnace.java.patch

```diff
@@ -75,14 +75,17 @@
           }
  
           if (!this.field_145850_b.field_72995_K) {
-@@ -346,12 +350,13 @@
-          return 0;
-       } else {
-          Item item = p_145952_0_.func_77973_b();
+@@ -342,16 +346,11 @@
+    }
+ 
+    private static int func_145952_a(ItemStack p_145952_0_) {
+-      if (p_145952_0_.func_190926_b()) {
+-         return 0;
+-      } else {
+-         Item item = p_145952_0_.func_77973_b();
 -         return func_201564_p().getOrDefault(item, 0);
-+         int ret = p_145952_0_.getBurnTime();
-+         return net.minecraftforge.event.ForgeEventFactory.getItemBurnTime(p_145952_0_, ret == -1 ? func_201564_p().getOrDefault(item, 0) : ret);
-       }
+-      }
++      return net.minecraftforge.common.crafting.FurnaceHelper.getItemBurnTime(p_145952_0_);
     }
```

Comment:
no need to remove all the logic here, it just bloats the patch. just patch the single return line

Reply:
like this ?